localization/english/event_localization/wc_draenor_events_l_english.yml

```
@@ -0,0 +1,21 @@
+﻿l_english:
+ WCDRA_5_TITLE:0 "The Dark Portal"
+ 
+ WCDRA_3_DESC:0 "Your army has reached the Dark Portal. After the long siege, your troops have occupied the surrounding area and are now awaiting your orders.\n\nFel energy forms in the portal's center, creating a window into the other world. Enemies can step through at any moment so you have to make a decision quickly: put an end to this accursed gateway once and for all or claim it for your own purposes."
+ WCDRA_3_OPT_A:0 "We must close it, now!"
+ WCDRA_3_OPT_B:0 "Our two worlds will stay connected."
+ 
+ WCDRA_5_DESC:0 "All the world trembles in the wake of this momentous day: the Dark Portal has been destroyed.\n\nBy bringing the best and the brightest of [portal_destroyer.GetFullNamePossessive] spellcasters to bear, the great stone gateway that served as a bridge between Azeroth and Draenor was torn asunder and obliterated to nothing more than a pile of rubble and an ominous rift; a tear in the fabric of space and time itself. As the dark energies of the portal exploded outward, the Black Morass - already corrupted by the Portal’s influence - has been scorched into an arid waste of dead earth that many are now beginning to call the Blasted Lands. No one is sure whether this damage is irreversible or if the land might someday be healed, and many magical masters are concerned by what further damage the rift might cause.\n\nAn even greater mystery, however, is what fate yet has in store for the people of Draenor who were caught on either side of the portal during its destruction. Those on Azeroth must now carve their own path in life and find a place in this world as best as they can, while those left behind on Draenor will be doomed to die a slow and painful death alongside their world, assuming nobody stabilizes the rift and reopens the portal…\n\n[portal_destroyer.GetFullNameNicknamed] will do what they can to ensure that doesn’t happen, of course, but what can be done if there yet remains someone with the power and will to open it from the other side?"
+ WCDRA_5_OPT_A:0 "We can never return home again..."
+ WCDRA_5_OPT_B:0 "What a waste..."
+ WCDRA_5_OPT_C:0 "And may it never open again!"
+ 
+ WCDRA_100_TITLE:0 "From a Dying World"
+ WCDRA_100_DESC:0 "Your guards bring you a [refugee.Custom('WomanManGirlBoy')]. They say that [refugee.GetSheHe] and [refugee.GetHerHis] group were caught by a patrol near the [refugee.Custom('GetDarkPortalRift')]. It would appear that they somehow got to the [refugee.Custom('GetPortalRift')] and crossed it, all without being noticed by guards on either side of the gateway.\n\nWhen you ask the fugitive, [refugee.GetSheHe] tells you [refugee.GetSheHe] had no other option but to sneak through the [refugee.Custom('GetPortalRift')]."
+ WCDRA_100_DESC_draenei:0 "The draenei have been brought to near-extinction by the Orcish Horde. Out of despair, [refugee.GetHerHis] group decided to come to the other side to avoid the genocide.\n\n"
```

Minor change, as I think this better fits the tone.
```suggestion
 WCDRA_100_DESC_draenei:0 "The draenei have been brought to near-extinction by the Orcish Horde. Out of despair, [refugee.GetHerHis] group decided to flee to Azeroth to avoid the genocide.\n\n"
```